mix.exs

```diff
@@ -48,6 +49,22 @@ defmodule Membrane.Opus.Plugin.Mixfile do
     ]
   end
 
+  defp dialyzer() do
+    opts = [
+      plt_local_path: "priv/plts",
+      flags: [:error_handling]
+    ]
+
+    if System.get_env("CI") == "true" do
+      # Store core PLTs in cacheable directory for CI
+      # For development it's better to stick to default, $MIX_HOME based path
+      # to allow sharing core PLTs between projects
+      [plt_core_path: "priv/plts"] ++ opts
```

Comment:
it seems different than in the template, why?

Reply:
Looks like I copied this from `with-native` branch from template plugin. Fixed.